docs/BackupAndRestore.md

```diff
@@ -0,0 +1,594 @@
+
+The backup jobs execution should be monitored on a regular basis to ensure that they complete successfully.
+By default, each backup job execution will be retried on failure which can be configured by the  `database.hotCopy.restartPolicy` setting.
+There are no backup retention policies currently available. The admin user is responsible to make sure that there is enough space in the backup volume of each SM for new hot copies to be created.
+The number of pods kept for failed backup jobs is defined by the `database.hotcopy.failureHistory` setting.
+This setting is useful when debugging failed backup job execution.
+The logs of the pod executing the job should be checked for errors.
+
+For example, the output below shows a failed full hot copy job:
+
+```bash
+kubectl get pods --selector job-name
+NAME                                                READY   STATUS             RESTARTS   AGE
+full-hotcopy-demo-cronjob-1613661840-rgpxv          0/1     Completed          0          32m
+full-hotcopy-demo-cronjob-1613663760-r8lm8          0/1     Error              1          14s
+```
+
+The logs from the pod indicate that there are no running SMs that match the provided labels.
+
+```bash
+kubectl logs full-hotcopy-demo-cronjob-1613663760-r8lm8
+Starting full backup for database demo on processes with labels 'backup cluster0 ' ...
+'backup database' failed: No SMs found matching the provided labels backup cluster0
+Error running hotcopy 1
+```
+
+Further investigation shows that the HC SMs statefulset has been scaled down to 0 replicas.
+
+```bash
+kubectl get statefulsets.apps
+NAME                                      READY   AGE
+admin-nuodb-cluster0                      1/1     3h41m
+sm-database-nuodb-cluster0-demo           2/2     3h41m
+sm-database-nuodb-cluster0-demo-hotcopy   0/0     3h41m
+```
+
+> **IMPORTANT**: It is important to take periodic journal hot copies when journal hot copy is enabled. Otherwise, the journal will keep growing which can lead to filling the archive volume space.
+
+It is possible to disable backup jobs creation by setting `hotcopy.enableBackups` to `false`.
+Custom backup jobs that use the `nuobackup` script can be configured to support more complex workflows and backup strategies.
+
+### Restore/Import source and type
+
+The restore/import source location can be one of the following:
+- remote _URL_ in a form of `protocol://authority/path`.
+The restore logic uses [cURL](https://curl.se/) to download a stream of data into an SM pod and restore the archive from it.
+Standard protocols such as _HTTP/s_, _FTP_ and _SFTP_ can be used as a source.
+Any other protocols that have built-in support in the shipped `cURL` binary in the NuoDB image can be used as well.
+Only basic authentication can be specified for remote sources. A remote location that has an authentication token embedded into the _URL_ can also be used.
```

Comment:
The second sentence sort of contradicts the "Only" part of the first. I would reword this as

> Basic authentication or an authentication token that is embedded in the URL are supported.

I guess this rules out certificate authentication or authentication other than basic auth that is passed as a request header (e.g. `Authorization: Bearer <token>`).

Reply:
Thanks! I will change it as suggested.